pages/login/index.js

```diff
@@ -30,64 +31,60 @@ export default function Login() {
 
   function updateLogin(e) {
     const { id, value } = e.target;
-    if (id != "checkbox") {
+    if (id !== "checkbox") {
       setLogin((prevValue) => {
         if (id === "email") {
           return { email: value, password: prevValue.password };
         }
+
         return { email: prevValue.email, password: value };
       });
     }
   }
```

Comment:
**suggestion:** I would recommend separating password and email. Makes it easier to read and cleans up the logic.